### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by egede**
Update to GSoC2021

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,40 +41,28 @@ As stated above everybody should attempt to complete this task
...
 2) Create a job in Ganga that demonstrates splitting a job into multiple pieces and then collates the results at the end.
-  - Use the included file `CERN.pdf`.
-  - In python, or through using system calls, split the pdf file into individual pages. 
-  - Create a job in Ganga that will count the number of occurences of the word "the" in the text of the PDF file. It should be counted whether it is capitalised or not. Make sure not to count other words with the same letters. So "The Ganga project is the best" should have a count of two, while "There is nothing here" should have a count of zero.
+  - Use the included file `LHC.pdf`.
+  - Create a job in Ganga that in python (or through using system calls) split the pdf file into individual pages. 
+  - Create a a second job in Ganga that will count the number of occurences of the word "it" in the text of the PDF file. It should be counted whether it is capitalised or not. Make sure not to count other words that have the the letters "it" inside them. So "It is best when it uses Ganga" should have a count of two, while "The initial test did no work" should have a count of zero.
```
Comment (alexanderrichards) [inline README.md]:
```suggestion
  - Create a a second job in Ganga that will count the number of occurences of the word "it" in the text of the PDF file. It should be counted whether it is capitalised or not. Make sure not to count other words that have the letters "it" inside them. So "It is best when it uses Ganga" should have a count of two, while "The initial test did no work" should have a count of zero.
```

File: README.md [hunk 2]
```diff
@@ -41,40 +41,28 @@ As stated above everybody should attempt to complete this task
...
+For this task we will not need to use the Ganga framework at all, you will need to show the following:
 
  - Ability to create a simple webserver using a python based web framework of your choosing
- - Have the webserver render dynamic content using a templating engine (e.g. jinja2)
- - Modify the server to allow updating of the content either by making it a RESTful service (with ajax requests client side) or by utilising WebSockets.
-
-
-(*__Note__ that the content for this task can be anything that you like and we would very much like to see you be as creative as possible with the actual design of the web page.*)
+ - Have the webserver render dynamic content using a templating engine (e.g. `jinja2`)
+ - Modify the server to allow updating of the content either by making it a RESTful service.
```
Comment (alexanderrichards) [inline README.md]:
```suggestion
 - Modify the server to allow updating of the content by making it a RESTful service.
```

**Review by alexanderrichards — CHANGES_REQUESTED** [review]
Looks good. The task descriptions are clearer I think now. Just some minor typos

**Review by alexanderrichards — APPROVED** [review]

**Comment by mesmith75** [comment]
Very nice. My only comment would be that for the GUI the technology is now pretty set by what is there. Do we want to require proficiency in that?

**Comment by egede** [comment]
> My only comment would be that for the GUI the technology is now pretty set by what is there. Do we want to require proficiency in that?

Can anybody include the buzz words to achieve that?

**Comment by alexanderrichards** [comment]
> > My only comment would be that for the GUI the technology is now pretty set by what is there. Do we want to require proficiency in that?
> 
> Can anybody include the buzz words to achieve that?

I suspect that simply requiring them to use the Flask framework would suffice. I'll put in a PR

**Pull request merged**